src/any_agent/serving/server.py

```diff
@@ -31,33 +33,49 @@ def _get_a2a_app(
 
 
 def _create_server(
-    app: A2AStarletteApplication, host: str, port: int, log_level: str = "warning"
+    app: A2AStarletteApplication,
+    host: str,
+    port: int,
+    endpoint: str,
+    log_level: str = "warning",
```

Comment:
Do you think this should be refactored to something like `**kwargs`? Might make it simpler as we expose more and more uvicorn configs (as our implementation matures).